config/settings/base.py

```diff
@@ -41,8 +41,9 @@ def safe_key() -> str:
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = env("DJANGO_SECRET_KEY", default=safe_key())
-HASHID_FIELD_SALT = env("DJANGO_HASHID_SALT", default=safe_key())
 DB_ENCRYPTION_KEY = env("DB_ENCRYPTION_KEY")
+HASHID_FIELD_SALT = env("DJANGO_HASHID_SALT", default=safe_key())
+HASHID_FIELD_ENABLE_HASHID_OBJECT = False  # Use plain strings
```

Comment:
I notice on Metecho we also have `HASHID_FIELD_ALLOW_INT_LOOKUP = True` -- do we want that here too?

Reply:
I don't think we use integer lookups anywhere, and can't remember why we added it to Metecho